tools/test/LcovMerger/java/com/google/devtools/lcovmerger/Main.java

```diff
@@ -67,6 +67,11 @@ public static void main(String[] args) {
       System.exit(1);
     }
 
+    if (flags.containsKey("filter_sources")) {
+      String[] regex = flags.get("filter_sources").split(",");
```

Comment:
What if a regex itself contains ","? Can you make `--filter_sources` a multistring flag?

Reply:
Do you mean `--filter_sources` being specified multiple times on the command line?

Reply:
Yes.

Reply:
Done. I had to refactor the flags handling logic so I moved it out to a new class `LcovMergerFlags`.